Trigger/CMakeLists.txt

```diff
@@ -38,4 +38,19 @@ cet_build_plugin(ReadTriggerInfo art::module
       Offline::TrackerGeom
 )
```

Comment:
Given that ReadTriggerInfo now depends on KinKal libraries shouldn't those be in the CMakeList as well?

Reply:
I updated the CMakeList to include these, but I'm not positive if I've used the correct syntax. Does it look right now?